packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -33,8 +33,19 @@ export interface IBasePickerProps<T> extends React.Props<any> {
   /**
    * A callback for what should happen when a person types text into the input.
    * Returns the already selected items so the resolver can filter them out.
+   * If used in conjunction with resolveDelay this will ony kick off after the delay throttle.
    */
   onResolveSuggestions: (filter: string, selectedItems?: T[]) => T[] | PromiseLike<T[]>;
+  /**
+   * A callback for when the input has been changed
+   */
+  onInputChanged?: (filter: string) => void;
```

Comment:
I'm very hesitant to add another onInputChange, there is already one that is used which is passed down to the autofill. Is there anyway that could be used instead?

Reply:
Yeah, I can remove this, not critical to our needs...

Reply:
Removed, thanks!